openai/completions.ts

```diff
@@ -43,26 +54,32 @@ AI: ${i.reply || ""}`
 export async function queryCompletionsChat(
   context: string,
   interactions: ChatInteraction[],
-  options: { apikey: string }
+  options: { apikey: string },
+  isGpt35Turbo = false
 ): Promise<ChatInteraction[]> {
   const completion = await queryCompletions(
```

Comment:
Here you should call `queryCompletions`/`queryCompletionsChat` depending on what model you are using. If you are using 3.5-turbo you are using chat, etc.